File: SumofDigits.c
//write a program to find sum of the digits

#include<stdio.h>  
 int main()    
{    
int n,sum=0,m;    
printf("Enter a number:");    
scanf("%d",&n);    
while(n>0)    
{    
m=n%10;    
sum=sum+m;    
n=n/10;    
}    
printf("Sum of the digits of the given number is=%d",sum);    
return 0;  
}   

File: SumOfDigits.java
import java.util.*;
class SumNumbers
{

public static void main(String args[])
{
    int a,sum=0;

    Scanner s = new Scanner(System.in);

    System.out.println("Enter a Number:");
    a = s.nextInt();

    while (a != 0) 
        { 
            sum = sum + a % 10; 
            a = a/10; 
        } 
System.out.println("The sum of the integers are:" + sum);
}
}
